--- scripts/ios-install-third-party.sh	2018-09-19 11:51:10.000000000 +0200
+++ scripts/ios-install-third-party.sh.new	2018-09-19 11:50:05.000000000 +0200
@@ -59,7 +59,7 @@

 mkdir -p third-party

-SCRIPTDIR=$(dirname "$0")
+SCRIPTDIR="$PWD/$(dirname $0)"

 fetch_and_unpack glog-0.3.4.tar.gz https://github.com/google/glog/archive/v0.3.4.tar.gz 69f91cd5a1de35ead0bc4103ea87294b0206a456 "\"$SCRIPTDIR/ios-configure-glog.sh\""
 fetch_and_unpack double-conversion-1.1.5.tar.gz https://github.com/google/double-conversion/archive/v1.1.5.tar.gz 96a8aba1b4ce7d4a7a3c123be26c379c2fed1def
