diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
index dd34c98..f406bbe 100644
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -299,6 +299,7 @@ afterEvaluate {
         // mergeResources task runs before the bundle file is copied to the intermediate asset directory from Android plugin 4.1+.
         // This ensures to copy the bundle file before mergeResources task starts
         def mergeResourcesTask = tasks.findByName("merge${targetName}Resources")
+        currentAssetsCopyTask.dependsOn(currentBundleTask) // See: https://github.com/facebook/react-native/issues/31157
         mergeResourcesTask.dependsOn(currentAssetsCopyTask)
 
         packageTask.dependsOn(currentAssetsCopyTask)
diff --git a/node_modules/react-native/scripts/ios-configure-glog.sh b/node_modules/react-native/scripts/ios-configure-glog.sh
index 9642c53..ffe1b40 100755
--- a/node_modules/react-native/scripts/ios-configure-glog.sh
+++ b/node_modules/react-native/scripts/ios-configure-glog.sh
@@ -20,6 +20,14 @@ if [ -z "$CURRENT_ARCH" ] || [ "$CURRENT_ARCH" == "undefined_arch" ]; then
     fi
 fi
 
+## @NOTE(gfanton): patch config.sub to work with apple silicon
+# download patched config.sub for darwin arm64
+curl -sSL --fail -o config.sub 'https://gist.githubusercontent.com/gfanton/f7dd19c299bef99d323cb08c650bb02e/raw/2f5a76ffd38cdac678adf6a9f12600e9d16e7345/config.darwin-arm-patch.sub'
+
+# check shasum of the file
+echo 'a743811464b00f3e2372f5368035fdc38f322a3a  config.sub' | shasum -a 1 -c || exit 1
+
+
 export CC="$(xcrun -find -sdk $PLATFORM_NAME cc) -arch $CURRENT_ARCH -isysroot $(xcrun -sdk $PLATFORM_NAME --show-sdk-path)"
 export CXX="$CC"
 
