diff --git a/node_modules/react-native-inappbrowser-reborn/android/build.gradle b/node_modules/react-native-inappbrowser-reborn/android/build.gradle
index 12c4628..c1fbb96 100644
--- a/node_modules/react-native-inappbrowser-reborn/android/build.gradle
+++ b/node_modules/react-native-inappbrowser-reborn/android/build.gradle
@@ -7,7 +7,7 @@ buildscript {
     if (project == rootProject) {
         repositories {
             maven { url "https://maven.google.com" }
-            jcenter()
+            mavenCentral()
             google()
         }
 
