diff --git a/node_modules/react-native/react.gradle b/node_modules/react-native/react.gradle
index dd34c98..f406bbe 100644
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -299,6 +299,7 @@ afterEvaluate {
         // mergeResources task runs before the bundle file is copied to the intermediate asset directory from Android plugin 4.1+.
         // This ensures to copy the bundle file before mergeResources task starts
         def mergeResourcesTask = tasks.findByName("merge${targetName}Resources")
+        currentAssetsCopyTask.dependsOn(currentBundleTask) // See: https://github.com/facebook/react-native/issues/31157
         mergeResourcesTask.dependsOn(currentAssetsCopyTask)
 
         packageTask.dependsOn(currentAssetsCopyTask)
